### PR TITLE
feat(info): Added Another Godmode Check And Health Info

### PR DESCRIPTION
**Pull request by TheGreenBandit**
i didnt use m_armor because that just returns 0,
maxhealth is the maximum health + armor they could have with their selected armor
normal health  is the armor + health of the player

again ignore previous commits i needa stop commiting on master branch

### CODE REVIEW

**Comment by TheGreenBandit** [comment]
Literally this was done in one commit i really do have commit on different branches lol

**Comment by TheGreenBandit** [comment]
Just had an idea, if armor > 0 i can include as an or statement too sctually, because menu devs who set the armor without knowing its unused will get detected

**Comment by Yimura** [comment]
Is this really needed? Have you ever encountered someone with unnatural health?

**Comment by TheGreenBandit** [comment]
> Is this really needed? Have you ever encountered someone with unnatural health?

No but if a mod menu has it, it would get detected

File: src/views/players/player/player_info.cpp [hunk 1]
```diff
@@ -142,6 +150,7 @@ namespace big
 				ImGui::Text("Money In Bank: %d", stats.Money - stats.WalletBalance);
 				ImGui::Text("Total Money: %d", stats.Money);
 				ImGui::Text("Rank: %d (RP %d)", stats.Rank, stats.RP);
+				ImGui::Text("Health: %d (MaxHealth: % d)", ped_health, ped_maxhealth);
```
Comment (Yimura) [inline src/views/players/player/player_info.cpp]: Does this line work? There's a space between the `%` and the `d`.

Comment (TheGreenBandit) [reply src/views/players/player/player_info.cpp:153]: Yeah works in game, ill fix the spelling mistake when i can

**Comment by TheGreenBandit** [comment]
Ready to commit

**Review by Yimura — APPROVED** [review]

**Pull request merged**